src/zui/ZUIPersonGridEditCell.tsx

```diff
@@ -201,7 +201,7 @@ const ZUIPersonGridEditCell: FC<{
                       <List
                         {...autoComplete.getListboxProps()}
                         subheader={
-                          <ListSubheader>
+                          <ListSubheader sx={{ position: 'relative' }}>
                             {suggestedPeople.length >= 0 &&
```

Comment:
The ListSubheader on line 185 also needs the same positioning! 
Also, I think there's a bit too much space above this header? I can't find exactly where this padding/margin is set, but it feels slightly off compared to other components